examples/vacationsApprovalAWSLambdas/.gitignore

```diff
@@ -0,0 +1,129 @@
+
```

Comment:
Instead of having a gitignore in each folder, wouldn't it be better to have just one gitignore at the root of the project?